index.bs

```diff
@@ -3815,55 +3830,36 @@ Working Group and Interest Group Notes</h3>
 
 		<li>
 			non-normative guides to good practices,
-
-		<li>
-			specifications where work has been stopped
-			and there is no longer consensus for making them a new standard.
 	</ul>
 
-	Some [=W3C Notes=] are developed through successive [=Working Drafts=],
-	with an expectation that they will become [=Notes=],
-	while others are simply [=published=].
-	There are few formal requirements to [=publish=] a document as a [=W3C Note=],
+	Some [=W3C Notes=] are developed through successive <dfn lt="Draft Note">Draft Notes</dfn>
+	before publication as a full [=Notes=],
+	while others are [=published=] directly as a [=Note=].
+	There are few formal requirements to [=publish=] a document as a [=W3C Note=] or [=Draft Note=],
 	and they have no standing as a recommendation of W3C
 	but are simply documents preserved for historical reference.
 
-	In order to publish a [=Note=],
+	In order to publish a [=Note=] or [=Draft Note=],
 	a [=Working Group=] or [=Interest Group=]:
 
 	<ul>
-		<li>
-			<em class="rfc2119">may</em> [=publish=] a [=Note=]
-			with or without its prior publication as a [=Working Draft=].
-
```

Comment:
It's not clear why this has been removed?

Reply:
This is meant to be an editorial rather than substantive change: the old text said how to publish a note, and noted that you could do so with or without prior publication as draft. The new text says how you can publish Notes or Draft Notes (same conditions), and that you can go from either to either. Having written that, it seemed that the bit of text you're inquiring about no longer adds anything. But there's nothing wrong with it either (other than s/Working Draft/Draft Note), so we could reinstate it if its absence makes things too unclear.

Reply:
Thanks, I wondered if that was the rationale @frivoal . I think the text does help to clarify that it is normatively permitted to go from WD -> Note or just publish a Note directly so I would not remove it as part of this change. But I don't think this is a major blocking concern.

Reply:
I think that when you're reading the diff, the removal of that text might be surprising, but if you rather look at the resulting text, it's already plenty clear that it is both allowed to go from Draft to Note or straight to Note, as we already say:

> 	Some W3C Notes are developed through successive Draft Notes
	before publication as a full Notes,
	while others are published directly as a Note.

So if you don't mind, I'd rather not reinstate that text.